string name
int32 time
string type
---
bool success
string info
